packages/core/src/types/contracts.ts

```diff
@@ -16,6 +16,11 @@ import type { ethers } from 'ethers'
 
 import type { IsNever, Join, NotEqual, Or } from './utils'
 
+// TODO: Removed during build for some reason so proxying here
+// and exporting from `src/internal.ts` for now
+// https://github.com/wagmi-dev/wagmi/issues/1306
+export type AbiParameter = AbiParameter_
```

Comment:
Need to investigate this at some point (probably a tsup thing), but this fixes the issue for now